### ISSUE TITLE
NVDA Fails to Show Drop-Down on Some Sites

### ISSUE THREAD

**Issue by ironmangary** [nvaccess/nvda#14500]
### Steps to reproduce:

1. In either Google Chrome or Mozilla Firefox, browse to careers.homedepot.com
2. Click "Create a New Account"
3. Scroll down to "State/Province"
 
### Actual behavior:

Where either a drop-down menu or combo box is expected, there is no way to access the State/Prfovince list. In screen review, it looks like this:

City
   This field is required. 
State/Province
  This field is required. 
Postal Code
   This field is required.  

Where "City" and "Postal Code" have text fields, "State/Province has nothing."

With focus mode turned on, Chrome reports nothing at all for State/Province, while Firefox reports "Text frame" but no action within this frame is reported.

### Expected behavior:

I would expect a drop down of states/provinces.  When viewing page source in Firefox, it is coded as such.  

I will attempt to attach the source of this page as en_US_registration.do.htm.zip.

#### NVDA version:

Version: 2022.3.3

#### Windows version:

OS Name Microsoft Windows 10 Pro
Version 10.0.19045 Build 19045

#### Name and version of other software in use when reproducing the issue:

Google Chrome Version 108.0.5359.125 (Official Build) (64-bit)

Mozilla Firefox 108.0.1 (64 bit)

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by seanbudd** [comment]
Have you reported this to the authors of the website? Where else have you seen this issue?

**Comment by ironmangary** [comment]
I have not seen this particular behavior anywhere else other than
through Home Depot's career website.  I am attempting to contact them
about it.  The form field was accessible by someone sighted, which is
why I wanted to bring it to your attention.

**Comment by seanbudd** [comment]
If it's just an issue with Home Depot, it is likely they have failed to make the site accessible.
I suggest the contact us page: https://corporate.homedepot.com/page/contact-us

**Comment by seanbudd** [comment]
Closing this as can't fix
